src/Workspaces/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexParser.cs

```diff
@@ -1271,7 +1271,7 @@ private void ParseCharacterClassComponents(ArrayBuilder<RegexExpressionNode> com
         private bool IsEscapedMinus(RegexNode node)
             => node is RegexSimpleEscapeNode simple && IsTextChar(simple.TypeToken, '-');
 
-        private bool TryGetRangeComponentValue(RegexExpressionNode component, bool isRight, out char ch)
+        private bool TryGetRangeComponentValue(RegexExpressionNode component, out char ch)
```

Comment:
Looks i accidently made a change here instead of in the regex PR.  However, it's a totally trivial change, so it shouldn't impact this PR going in.

Reply:
If I remember correctly, there is a corresponding corefx issue. Could you add a link if we have one?

---
In reply to: [210099883](https://github.com/dotnet/roslyn/pull/25520#discussion_r210099883) [](ancestors = 210099883)

Reply:
I don't know if there is one. @ViktorHofer you were tracking this right?  Did you ever open a bug on this strange behavior?

Reply:
The last i heard here:

```
 ViktorHofer on May 2 Member

This is definitely a bug and not by design. I would love this to be fixed but I fear that changing the existing behavior could break someone. I used a pattern searcher to go through some of our internal data and found 2 unique pattern which are built like that: [.-\-.*].
@ViktorHofer
ViktorHofer on May 2 Member

Ok correction, the two matches that I found aren't relevant here. It could definitely be the case that nobody ever used the defected behavior...
@CyrusNajmabadi
CyrusNajmabadi on May 2 •
Member

Ok cool. If you do get around to fixing it, let me now so i can keep roslyn up to date. Thanks! #Closed
```

Reply:
I opened https://github.com/dotnet/corefx/issues/31786 to track this since i couldnt' find an existing bug.

Reply:
i added this as a comment in the code in the regexFeatures PR.

Reply:
Did you find out the corefx issue?

---
In reply to: [210382497](https://github.com/dotnet/roslyn/pull/25520#discussion_r210382497) [](ancestors = 210382497,210099883)

Reply:
> Did you find out the corefx issue?

Could you clarify?  There's a bug filed against corefx about their bugger character class handling.  That bug is: https://github.com/dotnet/corefx/issues/31786

Was there something else you were referring to?

Reply:
Oops, sorry. You'd already commented with the link. Thanks